dist/Components/upperInfo/layout.jsx

```diff
@@ -55,15 +82,35 @@ class Layout extends React.Component {
   render() {
     return (
       <div>
-        <FontAwesomeIcon icon="home" />
-        <Type>Entire house</Type>
-        <Wrapper>
-          <Number>{this.state.guests} guests&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</Number>
-          <Number>{this.state.rooms} bedrooms&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</Number>
-          <Number>{this.state.beds} beds&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</Number>
-          <Number>{this.state.baths} baths</Number>
-        </Wrapper>
-        <Line />
+        <div>
+          <FontAwesomeIcon icon="home" />
+          <Type>Entire house</Type>
+          <Wrapper>
+            <span>{this.state.guests} guests&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span>
+            <span>{this.state.rooms} bedrooms&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span>
+            <span>{this.state.beds} beds&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span>
+            <span>{this.state.baths} baths</span>
+          </Wrapper>
+        </div>
+        <div>
+          <FontAwesomeIcon icon="map-marker-alt" />
+          <MidType>Great location</MidType>
+          <Inner>
+            <div>90% of recent guests gave this location a 5-star rating.</div>
```

Comment:
Is this fixed? Or comes from somewhere?  

I guess it's dummy for now.

Reply:
Yeah this is one of a few places where we could potentially cross reference data... not sure if they want us to pursue this or not.  In a real world scenario we would, but it would be a good question for Maia